src/components/suggest/_suggest_item.scss

```diff
@@ -62,7 +62,7 @@
 
   .ouiSuggestItem__label {
     @include ouiTextTruncate;
-    font-family: $ouiCodeFontFamily;
+    font-family: var(--oui-code-font-family);
```

Comment:
I don't see any issues tagged to the PR. Trying to understand the purpose behind this change. Why are we changing from sass to css variables?

Reply:
CSS variables will allow overriding them on the frontend, without having to load a new stylesheet.